yaml/buildings.yaml

```diff
@@ -18,6 +18,12 @@ globals:
       roof_orientation: {col: "roof:orientation"}
       addr_housenumber: {col: "addr:housenumber"}
       addr_street: {col: "addr:street"}
+  - &building_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail(building)
+  - &building_part_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail("building:part")
```

Comment:
Hmm.... does that mean someone added `building:part=dome` when they meant `roof:shape=dome`? If so, we should probably ignore the `building:part`. Looks like there's [22 instances](http://taginfo.openstreetmap.org/keys/building%3Apart#values) of `building:part=dome`, so it's likely a typo / mis-tag.

There's so few of these values (other than `yes` & `roof`) that perhaps I was wrong to suggest we include them in the list as well.